### PR TITLE
Fixed Navbar

### PR DESCRIPTION
**Pull request by tammaroivan**
## Summary 
- Fixed Navbar

## Screenshot
image

## Trellos 

https://trello.com/c/gmtTprjV/13-navbar

### CODE REVIEW

File: maquetado/scss/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.navbar {
+  align-items: center;
+  background: #FFF;
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: use variable colors

File: maquetado/scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  box-shadow: 0px 5px 12px 0px rgba(0,0,0,0.5);
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: same

File: maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  display: flex;
+  width: 100%;
+  height: 90px;
+  justify-content: space-between;
+  padding: 30px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 999999;
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: z-index :1

File: maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.navbar img {
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: dont use tags in html

File: maquetado/scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  height: 40px;
+  width: auto;
+}
+
+.navbar p {
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: same

File: maquetado/scss/styles.scss [hunk 6]
```diff
@@ -11,6 +11,7 @@ body {
 .main {
 	width: 70%;
 	margin: 0 auto;
+	margin-top: 70px;
```
Comment (FrankIglesias) [inline maquetado/scss/styles.scss]: use shorthand for margins

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: maquetado/scss/navbar.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
+.navbar {
+  align-items: center;
+  background: $white;
+  box-shadow: 0px 5px 12px 0px rgba(0,0,0,0.5);
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: use a variable color for the last param `rgba(....)`

File: maquetado/scss/navbar.scss [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  height: 90px;
+  justify-content: space-between;
+  padding: 30px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+}
+
+.navbar .logo {
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss]: nest logo and text inside `.navbar`

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by tammaroivan** [comment]
Not using tags in previous version (PR Fixes in navbar pr)

File: maquetado/scss/navbar.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  width: 100%;
+  height: 90px;
+  justify-content: space-between;
+  padding: 30px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;.
+
+  .text {
```
Comment (FrankIglesias) [inline maquetado/scss/navbar.scss:16]: I'd use another name to this class.. but it's ok for now

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**